client/controllers/suggestedIncidents.coffee

```diff
@@ -172,11 +174,18 @@ Template.suggestedIncidentsModal.helpers
     Template.instance().incidentCollection.find().count() > 0
   loading: ->
     Template.instance().loading.get()
+  annotatedCount: ->
+    if Template.instance().annotationCount.get()
+      count = Incidents.find({userEventId: Template.instance().data.userEventId}).fetch().length
```

Comment:
I might be a bit confused on terminology, but I think that it is the number of incidents.  The `count` variable is the number of records in the `counts` collection (referred to here as `Incidents`).  The number of suggested incidents is the `annotationCount` reactive var.  Does this sound correct, or am I counting the wrong things?